src/lib.rs

```diff
@@ -67,5 +67,5 @@ pub use crate::plan::selected_plan::{SelectedConstraints, SelectedPlan};
 pub use crate::mm::memory_manager;
 pub use crate::mmtk::MMTK;
 pub use crate::plan::{
-    Allocator, CopyContext, Mutator, MutatorContext, Plan, TraceLocal, TransitiveClosure,
+    AllocationSemantic, CopyContext, Mutator, MutatorContext, Plan, TraceLocal, TransitiveClosure,
```

Comment:
Why is this not `AllocationSemantics`?

Reply:
After a quick search, it seems there is no singular form for the word 'semantics'. I will change that.